proto/prysm/v1alpha1/BUILD.bazel

```
@@ -322,6 +351,7 @@ go_proto_library(
         "//proto/engine/v1:go_default_library",
         "//proto/eth/ext:go_default_library",
         "@com_github_golang_protobuf//proto:go_default_library",
+        "@com_github_offchainlabs_go_bitfield//:go_default_library",
         "@com_github_prysmaticlabs_go_bitfield//:go_default_library",
```

```suggestion
        "@com_github_offchainlabs_go_bitfield//:go_default_library",
```

Same here, seems problematic to have two bitfield deps?